Project-Lily/Project-Lily/DogListViewController.swift

```diff
@@ -65,7 +66,7 @@ class DogListViewController: UIViewController {
         case "DetailSegue":
```

Comment:
🍰 - Good opportunity to extract magic strings into an enum in a separate file. Having one place to store your storyboard's segue identifiers strings means you could call `case .detailSegue`: and have complete confidence that your strings will match up.

something like:
```
enum StoryboardIdentifiers: String {
   case detailSegue = "DetailSegue"
}
```

Reply:
Good Call! I am going to implement this for sure.